src/main/resources/db/migration/V0__initial_version.sql

```diff
@@ -0,0 +1,48 @@
+CREATE SCHEMA tasker;
+
+CREATE TABLE tasker.boards (
+                               "id" SERIAL PRIMARY KEY,
```

Comment:
from what I ready on the web, `UUID` sounds like better solution if we are concerned about security and consistency. It might have space and performance implications, but I don't think that it'll be an issue with this project.